frontend/zulu-app/.gitignore

```diff
@@ -0,0 +1,13 @@
+node_modules/**/*
```

Comment:
please make it one .gitignore (at the top of the github repo) - that is the standard as far as I recall.